server.js

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env node
```

Comment:
ah ok, didn't know that. It would make sense if this is a server, but since it isn't I vote for revert on this one